tar/untar_stream_test.ts

```diff
@@ -133,35 +133,31 @@ Deno.test("UntarStream() with size equals to multiple of 512", async () => {
 });
 
 Deno.test("UntarStream() with invalid size", async () => {
-  const readable = ReadableStream.from<TarStreamInput>([
-    {
-      type: "file",
-      path: "newFile.txt",
-      size: 512,
-      readable: ReadableStream.from([new Uint8Array(512).fill(97)]),
-    },
-  ])
-    .pipeThrough(new TarStream())
-    .pipeThrough(
-      new TransformStream<Uint8Array, Uint8Array>({
-        flush(controller) {
-          controller.enqueue(new Uint8Array(100));
+  const bytes = concat(
```

Comment:
We can now use [`toBytes()`](https://jsr.io/@std/streams/doc/unstable-to-bytes/~/toBytes) instead of `concat()` and `Array.fromAsync()`.